NPE in VertxEventBusMetrics on delivering message while handler registration is not finished yet

```
[ERROR] vert.x-worker-thread-28 - io.vertx.core.eventbus.impl.HandlerRegistration: Failed to handleMessage. address: some_address java.lang.NullPointerException
 at io.vertx.micrometer.impl.VertxEventBusMetrics.beginHandleMessage(VertxEventBusMetrics.java:90)
 at io.vertx.micrometer.impl.VertxEventBusMetrics.beginHandleMessage(VertxEventBusMetrics.java:33)
 at io.vertx.core.eventbus.impl.HandlerRegistration.deliver(HandlerRegistration.java:262)
 at io.vertx.core.eventbus.impl.HandlerRegistration.handle(HandlerRegistration.java:242)
 at io.vertx.core.eventbus.impl.EventBusImpl$InboundDeliveryContext.next(EventBusImpl.java:565)
 at io.vertx.core.eventbus.impl.EventBusImpl.lambda$deliverToHandler$5(EventBusImpl.java:524)
 at io.vertx.core.impl.ContextImpl.executeTask(ContextImpl.java:320)
 at io.vertx.core.impl.WorkerContext.lambda$wrapTask$0(WorkerContext.java:34)
 at io.vertx.core.impl.TaskQueue.run(TaskQueue.java:76)
 at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
 at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
 at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
 at java.lang.Thread.run(Thread.java:748)
```
```
[ERROR] vert.x-worker-thread-5 - io.vertx.core.impl.ContextImpl: Unhandled exception java.lang.NullPointerException
 at io.vertx.micrometer.impl.VertxEventBusMetrics.endHandleMessage(VertxEventBusMetrics.java:98)
 at io.vertx.micrometer.impl.VertxEventBusMetrics.endHandleMessage(VertxEventBusMetrics.java:33)
 at io.vertx.core.eventbus.impl.HandlerRegistration.deliver(HandlerRegistration.java:271)
 at io.vertx.core.eventbus.impl.HandlerRegistration.handle(HandlerRegistration.java:242)
 at io.vertx.core.eventbus.impl.EventBusImpl$InboundDeliveryContext.next(EventBusImpl.java:565)
 at io.vertx.core.eventbus.impl.EventBusImpl.lambda$deliverToHandler$5(EventBusImpl.java:524)
 at io.vertx.core.impl.ContextImpl.executeTask(ContextImpl.java:320)
 at io.vertx.core.impl.WorkerContext.lambda$wrapTask$0(WorkerContext.java:34)
 at io.vertx.core.impl.TaskQueue.run(TaskQueue.java:76)
 at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
 at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
 at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
 at java.lang.Thread.run(Thread.java:748)
```

--- a/src/main/java/io/vertx/micrometer/impl/VertxEventBusMetrics.java
+++ b/src/main/java/io/vertx/micrometer/impl/VertxEventBusMetrics.java
@@ -76,7 +76,7 @@ class VertxEventBusMetrics extends AbstractMetrics implements EventBusMetrics<Ve
 
   @Override
   public void handlerUnregistered(Handler handler) {
-    if (!handler.isIgnored()) {
+    if (isValid(handler)) {
       handlers.get(handler.address).decrement();
     }
   }
@@ -87,7 +87,7 @@ class VertxEventBusMetrics extends AbstractMetrics implements EventBusMetrics<Ve
 
   @Override
   public void beginHandleMessage(Handler handler, boolean local) {
-    if (!handler.isIgnored()) {
+    if (isValid(handler)) {
       pending.get(handler.address, Labels.getSide(local)).decrement();
       handler.timer = processTime.start();
     }
@@ -95,7 +95,7 @@ class VertxEventBusMetrics extends AbstractMetrics implements EventBusMetrics<Ve
 
   @Override
   public void endHandleMessage(Handler handler, Throwable failure) {
-    if (!handler.isIgnored()) {
+    if (isValid(handler)) {
       handler.timer.end(handler.address);
       if (failure != null) {
         errorCount.get(handler.address, failure.getClass().getSimpleName()).increment();
@@ -156,16 +156,16 @@ class VertxEventBusMetrics extends AbstractMetrics implements EventBusMetrics<Ve
   public void close() {
   }
 
-  public static class Handler {
+  private static boolean isValid(Handler handler) {
+    return handler != null && handler.address != null;
+  }
+
+  static class Handler {
     private final String address;
     private Timers.EventTiming timer;
 
     Handler(String address) {
       this.address = address;
-    }
-
-    boolean isIgnored() {
-      return address == null;
     }
   }
 }

